src/edu/csus/ecs/pc2/core/execute/IExecuteFrameNotify.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.execute;
+
+/**
+ * @author John Buck
+ *
+ */
+public interface IExecuteFrameNotify {
```

Comment:
The javadoc forthis interface should at least contain a brief summary of the purpose/intended usage of the interface.

Reply:
Fixed and pushed.